Cask

```diff
@@ -0,0 +1,9 @@
+;; -*- mode:lisp -*-
+
+(source gnu)
+(source melpa)
+
+(package-file "helm-swoop.el")
+
+(development
```

Comment:
You hadn't add `helm` as dependency :joy:

Reply:
Needless. Cask read dependency from the header of helm-swoop.el.
One piece of information should place one place.

Reply:
That's really good.